serverless/src/exportSearch/jsonToCsv.js

```diff
@@ -46,6 +46,11 @@ export const jsonToCsv = (jsonArray) => {
         const shortNames = collection[key].map((platform) => platform.shortName)
 
         collectionValues.push(JSON.stringify(shortNames.join(', '), replacer))
+      } else if (key === 'processingLevel') {
+        // Use the ID field in processingLevel
+        const { id } = collection[key]
+
+        collectionValues.push(`"${id}"`)
```

Comment:
```suggestion
        collectionValues.push(id)
```

Reply:
We need the quotes added here, to ensure that whatever comes back in the id field doesn't break the CSV delimiting.